dist/src/main/arctic-bin/conf/config.yaml

```diff
@@ -23,7 +23,11 @@ ams:
   #arctic.ams.cluster.name: default
   #arctic.ams.zookeeper.server: 127.0.0.1:2181,127.0.0.1:2182,127.0.0.1:2183
 
-# extension properties for like system
+  # kyuubi config
```

Comment:
Should we add default local terminal configurations here?

Reply:
local is default in code, does not need to be setted in conf file.